js/url.ts

```diff
@@ -71,12 +71,13 @@ export class URL {
     const searchParams = new urlSearchParams.URLSearchParams(this.search);
 
     for (const methodName of searchParamsMethods) {
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      /* eslint-disable @typescript-eslint/no-explicit-any */
       const method: (...args: any[]) => any = searchParams[methodName];
-      searchParams[methodName] = (...args: unknown[]): void => {
+      searchParams[methodName] = (...args: unknown[]): any => {
```

Comment:
Why `any` as you don't return a value?

Reply:
There is an assignability issue in 3.5.1.  Now TypeScript goes through each of the `methodName` values on `searchParams` and looks at the assignability, and some of those return things, so you can't assign a `void` to them.  I don't know if the code is actually wrong or not, but now TypeScript is doing some real valid analysis.  It is possible this method should really be returning something, but I just made a fix to get the code to pass under 3.5.1.